portfolio/src/main/webapp/style.css

```diff
@@ -10,4 +10,33 @@
 
 .invisible {
     display: none;
-}
\ No newline at end of file
+}
+
+.topnav {
+  background-color: rgb(0, 0, 0);
+  overflow: hidden;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  float: center;
+  color: #ffffff;
```

Comment:
nit: is #fff the same? If so it's preferred to use hexadecimal notation
https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation